scripts/gdal/gdal_helper.py

```diff
@@ -102,7 +102,7 @@ def run_gdal(
     if proc.stderr:
         get_log().warning("run_gdal_stderr", command=command_to_string(temp_command), stderr=proc.stderr.decode())
 
-    get_log().debug("run_gdal_succeeded", command=command_to_string(temp_command), stdout=proc.stdout.decode())
+    get_log().trace("run_gdal_succeeded", command=command_to_string(temp_command), stdout=proc.stdout.decode())
```

Comment:
this was spamming my console with the full gdal_info output, so I lowered the level, happy to raise it again if we really want it?

Reply:
That's a good call. The output formatting is also very bad when calling gdalinfo with json output